distribution/ecs/example/bastion.tf

```diff
@@ -0,0 +1,65 @@
+variable "bastion_public_key" {
+  description = "The public key used to connect to the bastion host. If empty, no bastion is created."
+  default     = ""
+}
+
+output "bastion_ip" {
+  value = var.bastion_public_key != "" ? aws_instance.bastion[0].public_ip : null
+}
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_security_group" "allow_ssh" {
+  count       = var.bastion_public_key != "" ? 1 : 0
+  name        = "qw_ecs_bastion_allow_ssh"
+  description = "Allow SSH inbound traffic from everywhere"
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```

Comment:
reading [this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group#removing-all-ingress-and-egress-rules), it looks like `egress  = []` is the more natural way to write that

Reply:
my understanding of the docs is that `egress = []` means "no egress rule", i.e "no egress allowed". The rule configured here is the exact opposite, it means "all egress allowed".

Reply:
![image](https://github.com/quickwit-oss/quickwit/assets/7913347/d32f03c9-467a-4871-9b43-9f0d6ef59fe9)